### DISCUSSION TITLE
GitHub Support does not respond after 5 days

### DISCUSSION THREAD

**Discussion by fernando-athena** [community/community#76541] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

It would actually be great to have something for urgent cases. Like priority queue based on type of case, etc. 

Anywho, thank you for seeing this post, I was deleting a project, and my whole company's github org was deleted along with all our repos; not sure why, and I don't care how at the moment because, well, I just need it back. Thankfully, we all have backups, but we need the org, teams, and repos reinstated because obviously version control is life. This is destroying our agility, and causing us lots of over collaboration; we could go to another version control temporarily, but obviously there are pipelines and whatnot triggering, so we're hoping this can work out.

Been stressed the hell out for the past 5 days, really could use some help here.

**ticket #2441899**

**Comment by hafiz-muhammad** [comment]
These are from GitHub Docs:
- [Restoring a deleted organization](https://docs.github.com/en/enterprise-server@3.11/admin/managing-accounts-and-repositories/managing-organizations-in-your-enterprise/restoring-a-deleted-organization)
- [Restoring a deleted repository that was owned by an organization](https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository#restoring-a-deleted-repository-that-was-owned-by-an-organization)

**Comment by leonasdev** [comment]
@fernando-athena 

Hello, may I ask if you eventually received assistance and successfully recovered all the relevant data? I'm facing a similar issue now, and I'm quite worried.
